Add convert to raw Flux option (#2584)
influxdata/docs-v2: content/influxdb/cloud/notebooks/create-notebook.md

    {{< nav-icon "notebooks" >}}
2. Click **+Create Notebook**.
3. Enter a name for your notebook in the **Name this notebook** field.
4. By default, a **Metric Selector** and **Visualization** cell appear (see [Cell types](/influxdb/cloud/notebooks/notebooks-overview/#cell-types) for details.)
5. Click the **+** icon to add a cell. See [Cell types](/influxdb/cloud/notebooks/notebooks-overview/#cell-types) for details on each type of cell.
6. Click **Preview** to preview the results of each cell in a raw data table without writing any data.
    {{% warn %}}
If your cell contains a custom script that uses any output function to write data to InfluxDB (the `to()` function) or send it to a 3rd party service, clicking **Preview** will write data.
    {{% /warn %}}
7. Click the dropdown menu next to **Preview** and select **Run** to show the results of each cell and write it to the specified output bucket.
8. Click the eye icon to hide a cell.

## View and edit Flux script in a notebook

> Convert your notebook cells into raw Flux script to view and edit it as code.

Reviewer: ```suggestion
Convert your notebook cells into raw Flux script to view and edit the code.
```

Reviewer: @mluu12 let's add this one to our August Cloud updates

1. In the navigation menu on the left, click **Notebooks** and click on your notebook to open it.

    {{< nav-icon "notebooks" >}}

> 2. Click the log icon under the cells you want to view as Flux.

Author: @drdelambre and @mnorriswalker -- Is this icon a log? It's not clear to me what it's supposed to be or what it has to do with Flux.

Select one of the following output cell-types:

- To recieve an alert, select [Notification](#notification) and complete the steps below.
- To write output to a bucket, select [Output to Bucket](#output-to-bucket) and complete the steps below.
- To create a task, select [Schedule](#schedule) and complete the steps below.

#### Notification

1. Enter a time range to automatically check the data and enter your query offset.
2. Customize the conditions to send an alert.
3. Select an endpoint to receive an alert:
   - Slack and a Slack Channel
   - HTTP post
   - Pager Duty
4. (Optional) Personalize your message. By default, the message is "Notification Rule: ${ r._notification_rule_name } triggered by check: ${ r._check_name }: ${ r._message }."
5. Click **Export as Task** to create your alarm.

#### Output to bucket

1. Select a bucket.
2. Click **Preview** to see what would be written to the bucket without commiting, or click **Run** in the upper left to write, or select **Export as Task** to schedule your output as a task.

#### Schedule

1. Enter a time and an offset to schedule the task.
2. Click **Export as Task** to save.

> 

Reviewer: ```suggestion
```

### View and edit Flux script in a cell
Convert your notebook cells into raw Flux script to view and edit the code. Conversion to Flux is available for the following cell types:
  - **Inputs: Metric Selector**
  - **Transform: Column Editor** and **Downsample**

> 1. Click the overflow menu icon under the cells you want to view as Flux, then select **Convert to |> Flux**. You won't be able to undo this step.

Reviewer: ```suggestion
1. Click the overflow menu icon in the cell you want to view as Flux, then select **Convert to |> Flux**. You won't be able to undo this step.
```

> 2. A Flux cell appears with underlying script for the selected cell.
> 3. View and edit the Flux script as needed.

Reviewer: ```suggestion
    A Flux cell appears with underlying script for the selected cell.
3. View and edit the Flux script as needed.
```

Reviewer: @noramullen1 looks good to me.
